pandas/tests/groupby/conftest.py

```diff
@@ -76,3 +76,12 @@ def three_group():
                       'D': np.random.randn(11),
                       'E': np.random.randn(11),
                       'F': np.random.randn(11)})
+
+
+AGG_FUNCS = ['sum', 'prod', 'min', 'max', 'mean', 'median', 'var', 'first',
+             'last', 'nth']  # TODO: ohlc?
+
+
+@pytest.fixture(params=AGG_FUNCS)
```

Comment:
we have these defined in pandas/contest.py, all numeric reductions; add something similiar, but you can use groupby in the name, the reason is that this can then easily be shared with resample tests for example.

also need a description. and a more descriptive name.

Reply:
Make sense, though given feedback on doing `nth` separately below I might just defer this for another PR